core/src/main/java/io/questdb/cairo/SecurityContext.java

```diff
@@ -71,7 +71,8 @@ public interface SecurityContext {
 
     void authorizeCreateGroup();
 
-    void authorizeCreateJwk(CharSequence userOrServiceAccountName);
+    default void authorizeCreateJwk(CharSequence userOrServiceAccountName) {
```

Comment:
do we need the `default`?

Reply:
there is subclass with identical method, which should be removed

Reply:
once we decided not to use default in SecurityContext, and removed all of them.
the reason was to avoid issues, that we forget to override the method in ReadOnlySecurityContext, for example, and the check will just pass blindly.

Reply:
Changed in latest commit